### ISSUE TITLE
Angular 6 Compat

### ISSUE THREAD

**Issue by agil-NUBBA** [doxiaodong/ng2-simplemde#17]
Hi,

Firstly, thanks for your module, works very well. From `npm install`

```
npm WARN ng2-simplemde@2.0.1 requires a peer of @angular/common@^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng2-simplemde@2.0.1 requires a peer of @angular/core@^5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng2-simplemde@2.0.1 requires a peer of @angular/forms@^5.0.0 but none is installed. You must install peer dependencies yourself.
```

There are no actually incompatibilities with Angular 6 as far as I've tested, could you remove the warning?

thanks in advance

**Comment by doxiaodong** [comment]
After today.

**Comment by doxiaodong** [comment]
fix in 2.0.2
